Package.swift

```diff
@@ -150,7 +150,7 @@ let package = Package(
     .package(
       name: "GoogleUtilities",
       url: "https://github.com/google/GoogleUtilities.git",
-      "7.3.0" ..< "8.0.0"
+      "7.4.0" ..< "8.0.0"
```

Comment:
@paulb777 @ryanwilson tagged GoogleUtils 7.4.0 for SwiftPM (on GoogleUtils repo) and am now pointing to GoogleUtils 7.4 in the Package manifest.